configuration.android.template

```diff
@@ -0,0 +1,14 @@
+// Replace IDSVR_BASE_URL placeholders to produce the configuration.android.jsx file
+const HaapiConfiguration = {
```

Comment:
Aim for business like configuration with only the most common settings.
Links to other resources (like the module README or fallback attestation) can explain other settings.